<ui version="4.0" >
 <class>AutoItems</class>
 <widget class="QDialog" name="AutoItems" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>AutoItems</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="spacing" >
    <number>3</number>
   </property>
   <property name="margin" >
    <number>3</number>
   </property>
   <item>
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QSplitter" name="splitter" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QTabWidget" name="tabWidget" >
        <property name="currentIndex" >
         <number>0</number>
        </property>
        <widget class="QWidget" name="tabDate" >
         <property name="geometry" >
          <rect>
           <x>0</x>
           <y>0</y>
           <width>241</width>
           <height>337</height>
          </rect>
         </property>
         <attribute name="title" >
          <string>Date</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_4" >
          <item>
           <layout class="QVBoxLayout" name="verticalLayout" >
            <property name="spacing" >
             <number>0</number>
            </property>
            <item>
             <widget class="QRadioButton" name="RBYears" >
              <property name="text" >
               <string>Years</string>
              </property>
              <property name="checked" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout" >
              <property name="spacing" >
               <number>0</number>
              </property>
              <item>
               <widget class="QSpinBox" name="SBYearFrom" >
                <property name="sizePolicy" >
                 <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="layoutDirection" >
                 <enum>Qt::RightToLeft</enum>
                </property>
                <property name="minimum" >
                 <number>-30000</number>
                </property>
                <property name="maximum" >
                 <number>30000</number>
                </property>
                <property name="value" >
                 <number>2008</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_8" >
                <property name="minimumSize" >
                 <size>
                  <width>21</width>
                  <height>21</height>
                 </size>
                </property>
                <property name="maximumSize" >
                 <size>
                  <width>21</width>
                  <height>21</height>
                 </size>
                </property>
                <property name="font" >
                 <font>
                  <pointsize>18</pointsize>
                  <weight>50</weight>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="text" >
                 <string>-</string>
                </property>
                <property name="alignment" >
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="SBYearTo" >
                <property name="sizePolicy" >
                 <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimum" >
                 <number>-30000</number>
                </property>
                <property name="maximum" >
                 <number>30000</number>
                </property>
                <property name="value" >
                 <number>2008</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QRadioButton" name="RBPeriod" >
              <property name="text" >
               <string>Period</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2" >
              <property name="spacing" >
               <number>0</number>
              </property>
              <item>
               <widget class="QDateEdit" name="DEPeriodFrom" >
                <property name="sizePolicy" >
                 <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize" >
                 <size>
                  <width>17</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="maximumSize" >
                 <size>
                  <width>1000</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="layoutDirection" >
                 <enum>Qt::RightToLeft</enum>
                </property>
                <property name="minimumDateTime" >
                 <datetime>
                  <hour>0</hour>
                  <minute>0</minute>
                  <second>0</second>
                  <year>1752</year>
                  <month>9</month>
                  <day>14</day>
                 </datetime>
                </property>
                <property name="currentSection" >
                 <enum>QDateTimeEdit::YearSection</enum>
                </property>
                <property name="displayFormat" >
                 <string>dd.MM.yyyy</string>
                </property>
                <property name="calendarPopup" >
                 <bool>true</bool>
                </property>
                <property name="date" >
                 <date>
                  <year>2008</year>
                  <month>6</month>
                  <day>22</day>
                 </date>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_9" >
                <property name="minimumSize" >
                 <size>
                  <width>21</width>
                  <height>21</height>
                 </size>
                </property>
                <property name="maximumSize" >
                 <size>
                  <width>21</width>
                  <height>21</height>
                 </size>
                </property>
                <property name="font" >
                 <font>
                  <pointsize>18</pointsize>
                  <weight>50</weight>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="text" >
                 <string>-</string>
                </property>
                <property name="alignment" >
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDateEdit" name="DEPeriodTo" >
                <property name="sizePolicy" >
                 <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize" >
                 <size>
                  <width>25</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="maximumSize" >
                 <size>
                  <width>1000</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="buttonSymbols" >
                 <enum>QAbstractSpinBox::PlusMinus</enum>
                </property>
                <property name="displayFormat" >
                 <string>yyyy.MM.dd</string>
                </property>
                <property name="calendarPopup" >
                 <bool>true</bool>
                </property>
                <property name="currentSectionIndex" >
                 <number>0</number>
                </property>
                <property name="date" >
                 <date>
                  <year>2008</year>
                  <month>11</month>
                  <day>19</day>
                 </date>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QCheckBox" name="CBDateFormat" >
              <property name="text" >
               <string>Set format</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3" >
              <item>
               <widget class="QLineEdit" name="LEDateFormat" >
                <property name="enabled" >
                 <bool>false</bool>
                </property>
                <property name="minimumSize" >
                 <size>
                  <width>150</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text" >
                 <string>yyyy.MM.dd</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer" >
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0" >
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QCheckBox" name="CBSplitPeriod" >
              <property name="text" >
               <string>Split by year and month</string>
              </property>
              <property name="checked" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="CBShowMonthName" >
              <property name="text" >
               <string>Show month name</string>
              </property>
              <property name="checked" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer" >
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabCounter" >
         <property name="geometry" >
          <rect>
           <x>0</x>
           <y>0</y>
           <width>264</width>
           <height>337</height>
          </rect>
         </property>
         <attribute name="title" >
          <string>Counter</string>
         </attribute>
         <layout class="QHBoxLayout" >
          <property name="spacing" >
           <number>3</number>
          </property>
          <property name="margin" >
           <number>3</number>
          </property>
          <item>
           <layout class="QVBoxLayout" >
            <property name="spacing" >
             <number>0</number>
            </property>
            <item>
             <layout class="QGridLayout" >
              <property name="verticalSpacing" >
               <number>9</number>
              </property>
              <item row="0" column="0" >
               <widget class="QLabel" name="label_7" >
                <property name="text" >
                 <string>Type</string>
                </property>
                <property name="buddy" >
                 <cstring>comboBoxCounterType</cstring>
                </property>
               </widget>
              </item>
              <item row="0" column="1" colspan="3" >
               <widget class="QComboBox" name="comboBoxCounterType" >
                <item>
                 <property name="text" >
                  <string>Decimal</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>Roman</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>Letters (Latin)</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>Letters (National)</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="1" column="0" >
               <widget class="QLabel" name="label_4" >
                <property name="sizePolicy" >
                 <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text" >
                 <string>Start</string>
                </property>
                <property name="buddy" >
                 <cstring>LECounterStart</cstring>
                </property>
               </widget>
              </item>
              <item row="1" column="1" colspan="3" >
               <widget class="QLineEdit" name="LECounterStart" >
                <property name="minimumSize" >
                 <size>
                  <width>70</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="toolTip" >
                 <string>1</string>
                </property>
                <property name="text" >
                 <string>1</string>
                </property>
                <property name="alignment" >
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="2" column="0" >
               <widget class="QLabel" name="label_5" >
                <property name="sizePolicy" >
                 <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text" >
                 <string>End</string>
                </property>
                <property name="buddy" >
                 <cstring>LECounterEnd</cstring>
                </property>
               </widget>
              </item>
              <item row="2" column="1" colspan="3" >
               <widget class="QLineEdit" name="LECounterEnd" >
                <property name="minimumSize" >
                 <size>
                  <width>70</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="toolTip" >
                 <string>10</string>
                </property>
                <property name="text" >
                 <string>10</string>
                </property>
                <property name="alignment" >
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="4" column="0" colspan="2" >
               <widget class="QLabel" name="label_3" >
                <property name="sizePolicy" >
                 <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text" >
                 <string>Length</string>
                </property>
                <property name="buddy" >
                 <cstring>spinBoxCounterLength</cstring>
                </property>
               </widget>
              </item>
              <item row="4" column="2" colspan="2" >
               <widget class="QSpinBox" name="spinBoxCounterLength" >
                <property name="minimumSize" >
                 <size>
                  <width>60</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize" >
                 <size>
                  <width>60</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="layoutDirection" >
                 <enum>Qt::RightToLeft</enum>
                </property>
                <property name="buttonSymbols" >
                 <enum>QAbstractSpinBox::PlusMinus</enum>
                </property>
                <property name="minimum" >
                 <number>1</number>
                </property>
                <property name="maximum" >
                 <number>100</number>
                </property>
                <property name="value" >
                 <number>1</number>
                </property>
               </widget>
              </item>
              <item row="5" column="0" colspan="3" >
               <widget class="QLabel" name="label_6" >
                <property name="sizePolicy" >
                 <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text" >
                 <string>Fill char</string>
                </property>
                <property name="buddy" >
                 <cstring>LECounterFillChar</cstring>
                </property>
               </widget>
              </item>
              <item row="5" column="3" >
               <widget class="QLineEdit" name="LECounterFillChar" >
                <property name="sizePolicy" >
                 <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize" >
                 <size>
                  <width>25</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="maximumSize" >
                 <size>
                  <width>25</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="text" >
                 <string>0</string>
                </property>
                <property name="alignment" >
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QCheckBox" name="CBLowerCase" >
              <property name="text" >
               <string>Counter lower case</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string>Prefix</string>
              </property>
              <property name="buddy" >
               <cstring>LECounterPrefix</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="LECounterPrefix" >
              <property name="minimumSize" >
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string>Suffix</string>
              </property>
              <property name="buddy" >
               <cstring>LECounterSuffix</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="LECounterSuffix" />
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>20</width>
                <height>51</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="layoutWidget" >
        <layout class="QVBoxLayout" >
         <item>
          <widget class="QTreeWidget" name="TWExample" >
           <column>
            <property name="text" >
             <string>Example</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="CBCreateFiles" >
           <property name="text" >
            <string>Create files</string>
           </property>
           <property name="checked" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="CBAsSubItems" >
           <property name="text" >
            <string>Add as subitems</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QPushButton" name="BTest" >
         <property name="text" >
          <string>Test</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="BApply" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>Apply</string>
         </property>
         <property name="checkable" >
          <bool>false</bool>
         </property>
         <property name="checked" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="BCancel" >
         <property name="text" >
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
